kube-client/src/client/node_proxy.rs

```diff
@@ -0,0 +1,71 @@
+use crate::{
```

Comment:
should we name this module `kubelet_debug` possibly? (i am not sure, just wondering if it makes more sense, haven't seen node_proxy around docs for this kubelet feature)

Reply:
I agree it's poorly documented. I've no issue renaming those to `kubelet_debug` even if I feel it's not only for debug, for example, `kubectl exec pod` wouldn't work if you set the flag to false as it's the same endpoint that the api server uses.

Reply:
Ah, interesting. I guess the most questionable requirement is the accessibility of the `kubelet` then.